### DISCUSSION TITLE
[Feature Request] - Playlist item composed of multiple tags (acts like AND filter)

### DISCUSSION THREAD

**Discussion by justin2380** [grafana/grafana#36972] [General]
If I use tags to search for a dashboard it filters them using the AND operator. For example I want tag "1205" and tag "CL5-D" so it only pulls back the dashboards that match those 2 tags. This makes it easy to find the dashboard that I want to look at on port that is having problems. 

What I would like to do is be able to do the same thing when creating a playlist. That way I can quickly glance at all the dashboards on that port without having to click into each one individually. The problem is the playlist is created by using the OR operator with tags. So when I chose "1205" and "CL5-D" I get all of 1205 and all of CL5-D. There is no easy way to create a playlist with multiple tags which is why we used them. 

Please let me know if I am missing something but I believe this would be a useful feature. Thanks!

Version: Grafana version: 3.0.1, commit: v3.0.1+2-g549eb15, build date: 2016-05-11 03:37:21
DataSource: InfluxDB
OS: Linux

This picture shows only dashboards matching those 2 tags.
![image](https://cloud.githubusercontent.com/assets/19711037/15748071/0972fb66-28a3-11e6-8e83-14d41336f406.png)
This picture will end up displaying all dashboards in my playlist that match 1205 and then display all the ones that match CL5-D so I end up getting everything.
![image](https://cloud.githubusercontent.com/assets/19711037/15748506/cf79b9d4-28a4-11e6-92e9-d589a33f0d5e.png)
![image](https://cloud.githubusercontent.com/assets/19711037/15748532/e303d976-28a4-11e6-85fe-063ff9bdaaa0.png)

![image](https://cloud.githubusercontent.com/assets/19711037/15748119/328f4400-28a3-11e6-95b2-ca8866321478.png)

**Comment by torkelo** [comment]
this is how it works now. You add a tag to the playlist. Currently you cannot add 2 or more tags as a single playlist item. currently each tag is its own playlist item, which is why it acts like an OR filter as they are evaluated in isolation

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!
